app/assets/stylesheets/content/ui_select.sass

```diff
@@ -1,6 +1,7 @@
 // keep this rule if you want ui-select drag-n-drop to work properly
-.ui-select-match .ui-select-match-item *
-  pointer-events: none
+.ui-select-match .ui-select-match-item
+  span, span *
+    pointer-events: none
```

Comment:
it block pointer events for all elements inside a span inside a match item inside the ui multiple select it order to not trigger leave and enter events during drag (AFAIR)